### ISSUE TITLE
[Feature]: Export kanban whole kanban board notes to one file

### ISSUE THREAD

**Issue by marekbrze** [mgmeyers/obsidian-kanban#586]
### Goal or desired outcome of this feature

Kanban boards are great for planning books. You can plan whole structure of the book using lists (parts of the book or chapters) and indivuidual notes (chapters or sections of the text). 

It woul be great to be able to export all the notes from the kanban board to a single file (one big markdown file or preferably .epub) to have the final text.

### Describe the feature

- One "export as" button on the kanban board
- All the notes merged into one big file based on the sequence on the kanban

### Screenshots, mockups, or videos

![Zrzut ekranu z 2022-07-09 15-19-55](https://user-images.githubusercontent.com/1033055/178107549-397049ce-9135-42ad-a0ac-9ba247a3ef95.png)

**Comment by AB1908** [comment]
I'm a little confused, isn't it already a markdown file?

**Comment by marekbrze** [comment]
@AB1908 contents of all the notes referenced in the kanban should be merged. 

So from the example above i would like to have a one long markdown/pdf/epub file with:

- Intro
  - Preface content
  - Introduction content
- Chapter 1
  - Section 1 content
  - Section 2 content
  - Section 3 content
- Chapter 2
  - Section 2 1 content
  - Section 2 2 content

**Comment by AB1908** [comment]
Hmm you can do this with some dataview, FWIW.

**Comment by marekbrze** [comment]
Yup, but drag'n'drop on canvas would make things much easier. It's superb for planning and outlining things
